### ISSUE TITLE
Multiple templates in one file

### ISSUE THREAD

**Issue by lpil** [michaeljones/gleam-templates#2]
Hello! Super cool tool 💜 

What would you think of having multiple templates compiled into one module? I think it would be nice to keep them together.

**Comment by michaeljones** [comment]
Good idea. I guess it could scope all templates together into one file and just name the render methods after the template file name. So like `user_profile` and `user_profile_builder` (for the one that outputs the stringbuilder if that is useful.)

**Comment by michaeljones** [comment]
@lpil - I've clearly forgotten about this. I'm happy to take a swing at it but only if you would still find it useful. 

I thought maybe a config entry in `gleam.toml` might work? To give the name of the final module? Or I guess all templates in a single folder could be collected together into a single `templates.gleam` module in that folder? Not sure. 

Happy to give it a go. Happy to close it :)

**Comment by lpil** [comment]
I don't have a pressing need, so perhaps we can just leave this open for anyone who finds it handy or interesting :)

**Comment by michaeljones** [comment]
Good to know. We can leave it open.
